src/Execution.ts

```diff
@@ -152,5 +152,5 @@ export function execute(
     debugObject.rawResults = results;
   }
 
-  return { rawResults: results, elmLibraries: elmJSONs, mainLibraryName: rootLibIdentifer.id };
+  return { rawResults: results, elmLibraries: elmJSONs, mainLibraryName: rootLibIdentifer.id, parameters: parameters };
```

Comment:
As of now, no. The only parameter for eCQMs is the Measurement Period. But it is possible that could change or this gaps code could get used in a manner outside of eCQMs where other parameters are expected.

I set it up as a parameter map because it is what is expected by the execution environment. We build that map before executing and I was hoping to use the same map for the re-calculations done in the filter parsing.

Reply:
That makes sense. Maybe we could just make a note via a comment or something in `parseQueryInfo` that the only supported parameter rn is measurement period

Reply:
Added a comment to the 'parseQueryInfo' that says the only supported parameter is "Measurement Period'. 4aecfea